api/handlers.go

```diff
@@ -17,7 +17,7 @@ var Collector = metrics.NewCollector()
 
 // Index serves a blank home page
 func Index(w http.ResponseWriter, req *http.Request) {
-	w.WriteHeader(http.StatusOK)
+	http.Redirect(w, req, "https://lbry.tv/", http.StatusSeeOther)
```

Comment:
by hardcoding the domain here we make it impossible for the project to be served elsewhere. perhaps if we really need to pass a URL we should have it in a config file

Reply:
good point, I'll make it a setting